modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -156,6 +158,11 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
       });
     });
 
+  const actOnType = (getAction: (type: string) => CombinedTargetsTableAction) => (_ui: boolean, args: Record<string, any>) =>
```

Comment:
Just curious, if `_ui` is not being used why is it in the method signature?

Reply:
Because it's the signature for command callbacks. We can't change that here.

Reply:
See https://github.com/tinymce/tinymce/blob/0f3b75bb87d37709c0be5a4d7c3d89463b74f716/modules/tinymce/src/core/main/ts/api/EditorCommands.ts#L37